CMakeLists.txt

```diff
@@ -6,12 +6,13 @@ include(CheckFunctionExists)
 include(CheckPrototypeDefinition)
 include(TestBigEndian)
 
-project(flint C)
+project(flint LANGUAGES C CXX)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake")
 
 find_package(GMP REQUIRED)
 find_package(MPFR REQUIRED)
+find_package(NTL REQUIRED)
```

Comment:
I'd make this optional as NTL with MPIR on windows 64-bit assumes LP64 and doesn't work.
Something like,
```
option(WITH_NTL "Build with NTL interface" OFF)
if (WITH_NTL)
    find_package(NTL REQUIRED)
endif()
```

Reply:
How does one check this ? Is it `cmake -D???` for some value of `???`.

Reply:
Yes, `cmake -DWITH_NTL=yes` would enable it.